src/app/security-group/sg-rules/sg-rules.component.scss

```diff
@@ -39,6 +39,10 @@ mat-form-field {
   text-align: left;
 }
 
+:host /deep/ .delete-button-cell {
```

Comment:
fix padding here to make it look better
![fixpadding](https://user-images.githubusercontent.com/2853979/32530168-14d28c9c-c46f-11e7-8ccb-b7f377e60380.png)